packages/ava/package.json

```diff
@@ -64,6 +64,7 @@
     "tslib": "^2.3.1"
   },
   "devDependencies": {
+    "@antv/algorithm": "^0.1.25",
```

Comment:
why is dev?

Reply:
graph deps it, but now we just move graph files, and have not finished it

I've @chenluli here
https://github.com/antvis/AVA/pull/505/files#diff-b631b9f3e6d513f3d244c081cbef910e7a7afa54c2c782ea607d48a6cbd76234R1